modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/policies-total-premium-due/index.css

```
@@ -0,0 +1,19 @@
+.total-premium-due-container .total-premium-due-title::after {
+	background-color: var(--primary);
+	content: '';
+	display: block;
+	height: 1px;
+	margin-top: 38px;
+	position: absolute;
+	width: 19px;
+}
+.total-premium-due-container .lexicon-icon {
```

Please, add a line break here.

```suggestion
}

.total-premium-due-container .lexicon-icon {
```

---

Done.